docs/manage/config-policies.rst

```diff
@@ -0,0 +1,179 @@
+
+.. tabs::
+
+   .. tab::
+
+      Cluster Config Policy
+
+      To set configuration policies at the cluster level, follow these steps:
+
+      #. Sign in to the Determined WebUI as a cluster administrator.
+      #. Navigate to **Config Policies**.
+      #. Choose **Experiments** or **Tasks** to display the editable configuration file.
+      #. Define the policies, then click **Apply**.
+      #. Confirm you want to apply the policies.
+
+   .. tab::
+
+      Workspace Config Policy
+
+      To set configuration policies at the workspace level, follow these steps:
+
+      #. Sign in to the Determined WebUI as a cluster or workspace administrator.
+      #. From your workspace, navigate to the **Config Policies** tab.
+      #. Choose **Experiments** or **Tasks** to display the editable configuration file.
+      #. Define the policies, then click **Apply**.
+      #. Confirm you want to apply the policies.
+
+CLI
+===
+
+Use the following commands to manage Task Config Policies via CLI:
+
+.. code:: bash
+
+   # List existing policies
+   det policy list
+
+   # Create a new policy
+   det policy create <policy_name> --config <policy_config_file.yaml>
+
+   # Update an existing policy
+   det policy update <policy_name> --config <updated_policy_config_file.yaml>
+
+   # Delete a policy
+   det policy delete <policy_name>
+
+*****************
+ Policy Examples
+*****************
+
+Here are some example policies:
+
+#. Limit resources.
+
+In the following example for an Agent resource manager (RM), the ``priority_limit`` is set to ``15``
+and ``max_slots`` is set to ``1``. This means a user cannot set a priority value lower than 15 and
+cannot set ``max_slots`` to greater than ``1``.
+
+.. note::
+
+   Tasks (NTSC - notebooks, Tensorboards, shells, and commands) and some resource managers (RMs)
+   have a priority. Priority levels are between 1 and 99. The default cluster priority is usually
+   42. The priority limit determines the order in which queued tasks run. In this example we are
+   using limits because for the agent resource manager, a lower number means a higher priority. For
+   a Kubernetes resource manager, a higher number means a higher priority.
+
+.. code:: bash
+
+   # Set priority limit to 15
+   constraints:
+      priority_limit: 15
+      resources:
+         max_slots: 1
+
+If your priority limits are above the defaults, for example if your notebook priority is set to 10
+and the limit is 15, then the request fails and the system displays a descriptive message.
+
+#. Limit maximum GPU usage per experiment:
+
+.. code:: yaml
+
+   name: max-gpu-limit
+   scope: cluster
+   config:
+     resources:
+       max_gpus: 4
+
+2. Set different priority limits for experiments and NTSC tasks:
+
+.. code:: yaml
+
+   name: priority-limits
+   scope: workspace
+   workspace: research
+   config:
+     scheduling:
+       max_priority:
+         experiments: 80
+         ntsc: 60
+
+3. Set default priority for all tasks:
```

Comment:
To set default priority for all tasks, enter the following in both experiment and task tabs in the global config policies page:

```
invariant_config:
  resources:
    priority: <desired priority value>
```